dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -18,6 +18,9 @@
 role.administrator=Administrator Role
 role.administrator.desc=A collection of users and/or groups assigned the server administrator role.
 
+role.viewer=Viewer Role
+role.viewer.desc=A collection of users and/or groups assigned the server viewer role.
```

Comment:
A collection of users, groups, or both that are assigned the server viewer role.